views/default/input/date.php

```diff
@@ -48,8 +52,20 @@
 	$vars['value'] = gmdate('Y-m-d', $vars['value']);
 }
 
+$datepicker_options = elgg_extract('datepicker_options', $vars);
```

Comment:
On 2nd thought that can wait. We need a big strategy to deprecate most of these instead of one by one